apps/dashboard/lib/Controller/DashboardController.php

```diff
@@ -103,8 +103,11 @@ public function index(): TemplateResponse {
 				'url' => $widget->getUrl()
 			];
 		}, $this->dashboardManager->getWidgets());
-		$configStatuses = $this->config->getUserValue($this->userId, 'dashboard', 'statuses', '{}');
+		// we avoid getting an empty array as it will not produce an object in UI
+		$configStatuses = $this->config->getUserValue($this->userId, 'dashboard', 'statuses', '{"weather": true}');
```

Comment:
```suggestion
		$configStatuses = $this->config->getUserValue($this->userId, 'dashboard', 'statuses', '');
```

Then we don't need to specify the default twice, since it is set below anyways if the default value cannot be parsed.